_src/pattern-library/sass/components/_forms.scss

```diff
@@ -0,0 +1,484 @@
+// ------------------------------
+// edX Pattern Library: Components - Forms
+
+// About: Contains base styling for forms
+
+// #GLOBAL
+// #INPUT TEXT
+// #INPUT RADIO/CHECKBOX
+// #SELECT
+// #TEXTAREA
+// #SLIDERS
+// #PROGRESS
+
+// ----------------------------
+// #GLOBAL
+// ----------------------------
+
+// sections of a form
+.form-group {
+    @include clearfix;
```

Comment:
NIT: please add "()" whenever you are calling a mixin (to help denote that the item called is a mixin.